### ISSUE TITLE
Problems with maven-enforcer-plugin and testcontainers 1.7.0

### ISSUE THREAD

**Issue by farodin91** [testcontainers/testcontainers-java#652]
I'm trying to integrate testcontainers into `janusgraph`. The build process checks the dependencies and maven-enforcer-plugin shows this error:
```
Dependency convergence error for org.jetbrains:annotations:15.0 paths to dependency are:
+-org.janusgraph:janusgraph-cassandra:0.3.0-SNAPSHOT
  +-org.testcontainers:testcontainers:1.7.0
    +-org.jetbrains:annotations:15.0
and
+-org.janusgraph:janusgraph-cassandra:0.3.0-SNAPSHOT
  +-org.testcontainers:testcontainers:1.7.0
    +-org.rnorth.duct-tape:duct-tape:1.0.6
      +-org.jetbrains:annotations:13.0
```

It would be cool to check the dependency consistency.

**Comment by bsideup** [comment]
Hi @farodin91,

Good to see janusgraph experimenting with Testcontainers 👍 

Thanks for letting us know! In addition to fixing it, I'll check if there are some `enforcer-`like plugins for Gradle to automate the check on our side.

FYI 1.7.1 was released on Friday with a bunch of fixes and it's recommended to update :)

**Comment by farodin91** [comment]
I updated the version but it doesn't fix the problem.

**Comment by bsideup** [comment]
Hi @farodin91,
we released 1.7.2 today where the problem is fixed, please try :)

**Comment by arnzel** [comment]
I get the error also in 1.7.3:

Dependency convergence error for org.jetbrains:annotations:15.0 paths to dependency are:
+-...
  +-...
    +-org.testcontainers:testcontainers:1.7.3
      +-org.jetbrains:annotations:15.0
and

+...
  +..
    +-org.testcontainers:testcontainers:1.7.3
      +-org.rnorth.duct-tape:duct-tape:1.0.7
        +-org.jetbrains:annotations:13.0

**Comment by arnzel** [comment]
Same error in 1.7.2

**Comment by kiview** [comment]
@farodin91 Do you also see the errors observed by @arnzel in your use case?
I'll reopen the issue.

**Comment by farodin91** [comment]
Seem to fail with the same issue by me.

**Comment by rnorth** [comment]
#772, which should fix this, was released in [1.8.1](https://github.com/testcontainers/testcontainers-java/releases/tag/1.8.1). Please could you check again?
